### ISSUE TITLE
Freezing when creating original dupes from printing pod

### ISSUE THREAD

**Issue by pslytely-psycho** [Cairath/ONI-Mods#30]
When a new item is requested from printing pod, game freezes and reports a mod error.

With only Lights out installed start a new game.  First print on the first cycle freezes game. Going to menu shows mod error. All menus are clickable, including return to menu where you get the error message.

 I am unaware of any logs that may apply.
This is the latest version downloaded from github as I was unsure if it successfully loaded from steam, although it appears it did as that is when the problem began with no problems prior while using this mod. It initially happened on the fourth cycle print on a heavily modded save. I started a new game and it crashed on the initial print. Only Lights Out was installed at the time.

In case I installed it wrong, I deleted everything in the original folder and copy/pasted the entire Github Lights Out folder contents to the folder. It appears to have compiled correctly upon looking at it again. I created the .json file and tried again with the same result.

I did not have this issue with the previous version.

**Comment by Cairath** [comment]
That does not sound like you installed it correctly - if I understood correctly that you replaced contents of the steam folder. Local mods need to be placed in /local folder. 
If you confirm you installed it in the correct directory (if you install it locally, please unsub on steam), please check your log. Look for `<<-- CaiLib -->> Loaded [ Lights Out ] with version 2019.9.15.7`

If you confirm the version in your log is the same as the quoted line, please attach your game log so I can analyze it. Without a log I can't and will not do anything.

`C:\Users\<username>\AppData\LocalLow\Klei\Oxygen Not Included\output_log.txt`

**Comment by Cairath** [comment]
(this sounds like the crash I fixed this morning)

**Comment by pslytely-psycho** [comment]
Ah, I have not installed a /local mod on ONI before. That is why I tried to be complete in my description. I will only ask one question then, where exactly is the /local file for ONI? I assumed it meant the file containing the .exe, but that didn't appear to work. I am not a programmer, but I have screwed with .config files on many games over the last decade, so I get it that I screwed the pooch, and won't bother you with a .log that will only say I screwed up. 
I didn't realize how many of your mods I use, they've all been great thus far! Kudos!

**Comment by pslytely-psycho** [comment]
[output_log.txt](https://github.com/Cairath/ONI-Mods/files/3613781/output_log.txt)
Interesting. As I haven't yet figured out /local installation, I re-subbed the mod and ran the game. The log doesn't seem to have the line showing your mod loaded although it's dark. So I think Steam/ONI are not playing nice. I wanted to find out the version I was getting off Steam, unfortunately no joy there. Searching in Notepad++ shows no mod loaded. So I assume it's not the updated version.

**Comment by Cairath** [comment]
That looks like a log from a restart of the game when they were disabled -- the log resets every time you restart the game, so ideally the one with info would be after you load the game with mods *enabled* or the crash occurs, before you restart the game.

Local mod installation is described in the readme of this repository:  https://github.com/Cairath/ONI-Mods#download - you create the local folder next to the steam folder -  but remember that you need to unsubscribe the steam version so it doesn't load it twice

**Comment by pslytely-psycho** [comment]
2019-9-15-3
I re-subbed and added Mod Manager.
Mod Manager gave me the version I have installed. That log looks like nothing is loaded, but I assure you Lights Out was working as it was dark. Looks like Steam's not downloading the newest version.
Thanks for your patience.
Off to do it right.
Thanks for the quick response, I appreciate it greatly.
Peace out!

**Comment by pslytely-psycho** [comment]
C:\Users\pslyt\Documents\Klei\OxygenNotIncluded\mods\local
copied downloaded Lights Out file to here, ran the game three times to be sure, does not seem to be working. Did I again miss something important?

**Comment by Cairath** [comment]
It looks like it is loading the dll, but the mod does not look to be enabled. Have you remembered to re-enable it in the menu after you installed it locally?

**Comment by pslytely-psycho** [comment]
[output_log.txt](https://github.com/Cairath/ONI-Mods/files/3613858/output_log.txt)
I screwed up. I was experimenting with subbing/unsubbing and forgot to unsub before testing. LO is not showing up in my mod menu. Here is the proper log. My apologies.

**Comment by Cairath** [comment]
The log still mentions subscribing to the mod on steam.

This is how the folder structure should look like.
![image](https://user-images.githubusercontent.com/10224213/64923298-e8ca7180-d7d8-11e9-94b9-e0832bc9d4ff.png)

![image](https://user-images.githubusercontent.com/10224213/64923316-10b9d500-d7d9-11e9-967a-780d039c584f.png)


If you placed the mod folder correctly then I am out of ideas what else could have been done wrong.

**Comment by pslytely-psycho** [comment]
OK, I got it.
I used the steam downloader to download it and put those files in the local/Lights Out file after deleting the files there. Perhaps a corrupted download? Sorry to take up so much of your time this weekend. Your assistance was invaluable, and I learned quite a bit. It's a bit different than manually modding Skyrim, GTAIV and V, Civilization or Kerbal Space Program. Seems like each one is a new adventure.
You rock man. Thanks.
P.S. it was definitely unsubbed and restarted for that log I sent last. Don't know why it would show as still subbed. It did not appear in my mods tab, nor was it working in game.
